PublishTestResultScreenshotV0/index.ts

```diff
@@ -37,26 +45,76 @@ async function run() {
 
 run();
 
-async function uploadScreenshots(failedTests: ShallowTestCaseResult[]) {
+/**
+ * Given system [build or release] returning the Id of the latest TestRun 
+ * @param system 
+ * @returns Latest TestRun Id
+ */
+async function getTestRunIdBy(system : string): Promise<number> {
+    let ret :number = -1;
+    let testRuns : TestRun[] = [];
+    let today = new Date();
+    let yesterday = new Date();
+    yesterday.setDate(today.getDate() -1);
+
+    let buildid = tl.getVariable("BUILD_BUILDID");
```

Comment:
For me this is not working, it would return an alphanumeric value when the task is executed in a release pipeline e.g. `1ab34abdcd272028g859e9375e99192b6eca4490` and it cannot then be converted to an array of numbers needed by the `queryTestRuns` function. I therefore end with error message from the task
```
2021-07-22T14:08:15.8159480Z ##[error]buildIds should be comma seperated list of valid Ids.
2021-07-22T14:08:15.8161630Z ##[debug]Processed: ##vso[task.issue type=error;]buildIds should be comma seperated list of valid Ids.
2021-07-22T14:08:15.8165370Z ##[debug]Processed: ##vso[task.complete result=Failed;]buildIds should be comma seperated list of valid Ids.
```

Reply:
It seems, simply removing the buildId param and sending `undefined` when `SYSTEM` is `release` actually fixes it.

Reply:
hmmmm, in my setup, release pipeline is triggered by a success build. That's why BuildId is available when I am testing the code. I will verify if removing BuildId also work later.